CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `BasicKernel` and `CompositeKernel` base classes
 - Activated `pre-commit.ci` with auto-update
 - User guide for active learning
+- Documentation building now uses a lockfile to prevent secondary dependency changes
```

Comment:
```suggestion
- Documentation building now uses a lockfile to fix the exact environment
```

i think secodnary envs are part of that but we dont even have the exact primary ones without the lock right?

Reply:
Yeah, good point